### DISCUSSION TITLE
Duplicate SQL query user_history in admin/user.deleteUser()

### DISCUSSION THREAD

**Discussion by jzmrsh** [opencart/opencart#12530] [General]
Apparently this is harmless.
![image](https://github.com/opencart/opencart/assets/53591150/d3f68e3f-6bac-46f2-b7f3-8002912b03e3)

**Comment by felipoantonoff** [comment]
It's really redundant SQL, it won't hurt, but it's unnecessary.

I think maybe the notice as an Issue would be better.
